holidays/countries/saint_vincent_and_the_grenadines.py

```
@@ -0,0 +1,99 @@
+)
+
+
+class SaintVincentAndTheGrenadines(ObservedHolidayBase, ChristianHolidays, InternationalHolidays):
+    """Saint Vincent and the Grenadines holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Saint_Vincent_and_the_Grenadines>
+        * <https://www.timeanddate.com/holidays/saint-vincent-and-the-grenadines/>
+        * <https://svgtourism.com/plan-your-trip/public-holidays/>
+        * <https://www.gov.vc/index.php/government/laws-of-svg>
+        * <https://www.timeanddate.com/holidays/saint-vincent-and-the-grenadines/national-spiritual-baptist-day>
```

Pylint (3.3.7)

[convention] Line too long (112/100)

(C0301)